### DISCUSSION TITLE
The solution to Dual Source Blend issue

### DISCUSSION THREAD

**Discussion by kciok** [ChefKissInc/NootedRed#294] [Q&A] [answered]
Aloha there!

Firstly please allow me to express my gratitude to this KEXT for providing me with a Hackintosh experience on my AMD processor!

But it seems that I am currently unable to find a solution for handling the Dual Source Blend issue. Could there be kind souls who would like to explain how one should handle this, or share a link to an article that has provided solutions?

**Comment by Startive** [comment]
There are no fixes at the minute, hopefully it gets fixed soon but it seems like this kext is getting kinda inactive unfortunately

**Reply by VisualEhrmanntraut** [reply]
I have not disappeared.

**Comment by VisualEhrmanntraut** [comment, marked as answer]
https://chefkissinc.github.io/applehax/nootedred/#frequently-asked-questions

**Reply by VisualEhrmanntraut** [reply]
There are some workaround here for the time being.
